src/components/Select/Select.scss

```diff
@@ -150,6 +150,11 @@ $stacking-order: (
 
 .newDesignLanguage {
   .Backdrop {
+
```

Comment:
This is an annoying bug in stylelint ([i think it's been fixed in latest](https://github.com/stylelint/stylelint/releases/tag/13.3.3) but we need to bump). Revert this.

Reply:
`sewing-kit format` did it automatically but linting failed because of it hahaha, I already pushed a new commit removing it

Reply:
Yeah it sucks, when fixing linter errors stylelint doesn't actually ignore ignored blocks which, uh, yeah :(

Gotta update SK to pull that fix in at some point.